docs/CLI/index.md

```diff
@@ -61,3 +63,12 @@ docker run -it --rm -v $(pwd):/workspace ghcr.io/oras-project/oras:v0.12.0 help
 ```
 
 > Note: the default WORKDIR  in the image is `/workspace`.
+
+## Verify 
+
+```
```

Comment:
If you add bash or shell to the top of these blocks the template should better format the syntax (e.g syntax highlighting). Although we’d probably want it to be consistent for command blocks across the page!

Reply:
@vsoch Good catch. I have updated it according to your suggestion. Thanks!